pkg/runit/servicebuilder.go

```diff
@@ -74,6 +74,7 @@ var DefaultBuilder = &ServiceBuilder{
 }
 
 var DefaultChpst = "/usr/bin/chpst"
+var DefaultContain = "/usr/bin/contain"
```

Comment:
Apparently I hallucinated that we had extracted that path to a constant. Yeah, ideally we allow flat configuration (clients may list out these dependencies) but that their defaults are kept isolated to their packages to maintain high cohesion (only `runit` should "know" about chpst, only XXX should no about `nolimit` and `contain`, etc)